src/main/resources/reference.conf

```diff
@@ -156,6 +156,7 @@ resourceTypes = {
           controlled-user-private-workspace-resource = ["deleter"]
           controlled-application-shared-workspace-resource = ["writer", "reader"]
           wds-instance = ["owner"]
+          kubernetes-app = ["creator", "manager"]
```

Comment:
I think we only want to give workspace owners `manager` role on kubernetes-app. Similar to `google-project`: https://github.com/broadinstitute/sam/blob/develop/src/main/resources/reference.conf#L608-L614

So here:
```
kubernetes-app = ["manager"]
```